lib/util/eventobserver.js

```diff
@@ -56,15 +57,16 @@ class EventObserver extends EventEmitter {
         const publisherPayload = Object.assign(payload ? payload : {}, { elapsedTime, level });
         this._publisher.publish(group, name, publisherPayload);
       }
+      const event = Object.assign(payload ? { payload } : {}, {
+        elapsedTime,
+        group,
+        level,
+        name,
+        timestamp
+      });
 
+      log.info('event', event);
```

Comment:
Since logger requires a minimum of `info` level, should we mention this in the changelog and documentation?

Reply:
yes, I have added a comment about this in the example.